src/main/markdown/gettingstarted.md

```diff
@@ -1,119 +1,154 @@
 Getting Started
 ===
 
+Starting with 2.11.0 of the GWT SDK the webAppCreator is deprecated and will be removed in a future version. Instead, 
+we will use the [gwt-maven-archetypes](https://github.com/tbroyer/gwt-maven-archetypes) by Thomas Broyer to create a new
+GWT project. Using the **artifactId: modular-webapp** the plugin generates a Maven project with separate modules for
+
+* client
+* server
+* shared.
+
+Separation of code is the way to go in modern web development. 
+
+## TOC
+
 *  [Prerequisites](#prereqs)
-*  [Download and Install GWT](#download)
 *  [Create your first web application](#create)
-*  [Run locally in development mode](#run)
+*  [Run locally in super dev mode](#run)
 *  [Make a few changes](#change)
+*  [Debugging the browser](#bebug)
 *  [Compile and run in production mode](#compile)
 *  [Set up an IDE](#setup)
 
 ## Prerequisites<a id="prereqs"></a>
 
-1.  You will need the Java SDK version 1.8 or later. If necessary, download and
-    install the Java SE Development Kit (JDK) <a
-    href="http://www.oracle.com/technetwork/java/javase/downloads/index.html" 
-    rel="nofollow">from Oracle</a> or <a href="https://adoptopenjdk.net/" 
-    rel="nofollow">from Adopt OpenJDK</a> for your platform.
-2.  Apache Ant is also necessary to run command line arguments in this sample. If
-    you don't already have it, install <a href="http://ant.apache.org/" rel="nofollow">Apache Ant</a>.
+1.  You will need the Java SDK version 11 or later. If necessary, download and
```

Comment:
GWT 2.11 still supports running and building in Java 8, but we encourage newer builds.